styleguide/components/Footer/index.js

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import Version from 'rsg-components/Version';
 
 import theme from 'styleguide/theme';
-import { bodySmall, headingLarge } from 'themes/mixins';
+import { heading } from 'styles';
```

Comment:
This is so much better. I wonder if just "styles" is too vague though? What about things like, say, focus styling? (Which will definitely be approached in a similar way to type styles here, and then applied to multiple components...)

Reply:
You _can_ import from more specific folders/modules (`import { heading } from 'styles/typeTokens';` would work here). But I think having one consistent entrypoint is a nicer developer/designer experience. You can _store_ those focus styles in `styles/focus.js` (and should!), but you don't have to _import_ them that way in usage.

That's actually what's happening already here; check out `src/styles` to see how it works.

Reply:
Ohhhhhhh I getcha. Coolio, let's do it your way.